src/bp-groups/bp-groups-cache.php

```diff
@@ -319,17 +317,15 @@ function bp_groups_clear_user_group_cache_on_other_events( $user_id, $group_id )
  *   - A group's metadata is modified.
  *
  * @since 2.7.0
- *
- * @return bool
  */
 function bp_groups_reset_cache_incrementor() {
-	return bp_core_reset_incrementor( 'bp_groups' );
```

Comment:
The main change here is removing returned values from side-effect-only functions.